tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -37,4 +37,33 @@ Check Image Build Status
   Run Keyword If    "${build_status}" == "${target_status}"
   ...    Log   The '${build_name}' image match the expected target status
   ...  ELSE
-  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
\ No newline at end of file
+  ...    Fail  The '${build_name}' image build status doesn't match the target status '${target_status}'
+
+Get Last Build
+    [Documentation]    Returns last build
+    [Arguments]    ${namespace}    ${word}
+    ${build} =    Run    oc get builds -n ${namespace} | grep ${word} | awk '{print $1;}'
+    [Return]    ${build}
+
+Delete Build Using Name
+    [Documentation]    Delete the build
+    [Arguments]    ${namespace}    ${build_name}
+    Run    oc delete builds ${build_name} -n ${namespace}
+    Sleep    120s
+    ${build} =    Run    oc get builds -n ${namespace} | grep ${build_name} | awk '{print $1;}'
+    Should Not Be Equal    ${build}    ${build_name}
+
+Start New Build
+    [Documentation]    Start new Buid
+    [Arguments]    ${namespace}    ${buildconfig}
+    ${name} =    Run    oc start-build ${buildconfig} -n ${namespace}
+    @{list_for} =    Split String    ${name}    ${SPACE}
+    @{name_list} =    Split String    ${list_for}[0]    /
+    [Return]    ${name_list}[1]
+
+Get Build Status By Name
```

Comment:
I think `Get Build Status` is enough

Reply:
`Get Build Status` is already present in the build file
so that's why I used `Get Build Status By Name`

Reply:
is there a reason to create a new keyword to get status if there's already one available?

Reply:
Yes
The `get status` keyword uses a web browser.
But for our test case, we are using OpenshiftCLI or oc commands that's why we have to create another keyword that work with openshift or oc..

Reply:
hm okay, then I'd change the name of the kw. I think the best approach would be to rename the already existent kw to `Get Build Status From UI` but in that case you should change accordingly all the TCs using that keyword (I think it should be only one).
As alternative, you could call you keyword `Get Build Status From Oc`. @jgarciao @rkamble5 what do you prefer?

Reply:
`Get Build Status From Oc` I think this one is better/good